### ISSUE TITLE
As citizen I want a dashboard over Covid-19 data, to track the status.

### ISSUE THREAD

**Issue by ohnewein** [noi-techpark/odh-web-components#17]
Similar to Trento:

https://www.linkedin.com/feed/update/activity:6647484135663374336

Ideally it is a web component, which can be inserted into any website.

**Comment by ohnewein** [comment]
Maybe it is possible to reuse rebuild the dashboard of Trento, bit connect to data inserted by @bertolla ?

**Comment by Piiit** [comment]
@ohnewein Yes that would be very nice. Would love to work on that... next sprint? Is it possible to get the source code of https://covid19trentino.fbk.eu/?

**Comment by ohnewein** [comment]
@Piiit It is a tool developed by Maurizio Napolitano. We can ask him. But I assume it is a JS frontend, isn't it?
